maxMoviesInYear.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @name maxMoviesInYear
+ * @description Function to find years with maximum number of movies licensed
+ * @param {Array} moviesList The list of movies
+ * @returns {Number[]} Years with max movies on platform
+ */
+function maxMoviesInYears(moviesList) {
+    if (moviesList.length === 0) return [];
```

Comment:
Do we need this line?

The more logic you don't  branch off, the better you prevent the cognitive overload